app/views/shared/_recent_bp_log.html.erb

```diff
@@ -7,7 +7,7 @@
     <% if display_model == :facility %>
       by healthcare workers at <%= @region.name %>
     <% elsif display_model == :user %>
-      by this healthecare workers.
+      by this healthcare worker
```

Comment:
Did we want to lose the period?  Approving to unblock.

Reply:
Yup :) intentionally removing the period